<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VsUserAccount</class>
 <widget class="QWidget" name="VsUserAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>361</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEditPortalName">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Portal eg http://myportal.com</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditUser">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>User</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditPassword">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
